content/blog/tidymodels-2021-q2/index.Rmd

```diff
@@ -0,0 +1,232 @@
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. Earlier this year, we [started regular updates](https://www.tidyverse.org/blog/2021/03/tidymodels-2021-q1/) here on the tidyverse blog summarizing recent developments in the tidymodels ecosystem overall. You can check out the [`tidymodels`](https://www.tidyverse.org/tags/tidymodels/) tag to find all tidymodels blog posts here, including those that focus on a single package or more major releases, but the purpose of these roundup posts is to keep you informed about any releases you may have missed and useful new functionality as we maintain these packages.
+
+Since our last roundup post, there have been 19 CRAN releases of 15 different packages. That might sound like a lot of change to absorb as a tidymodels user! However, we purposefully write code in small, modular packages that we can release frequently, to make models easier to deploy and our software easier to maintain. You can install these updates from CRAN with:
+
+```{r, eval = FALSE}
+install.packages(c("broom", "butcher", "discrim", "hardhat", "parsnip", "probably", 
+                   "recipes", "rsample", "stacks", "themis", "tidymodels", 
+                   "tidyposterior", "tune", "workflowsets", "yardstick"))
+```
+
+The `NEWS` files are linked here for each package; you'll notice that many of these releases involve small bug fixes or internal changes that are not user-facing. It's a lot to keep up with and there are some super useful updates in the mix, so read on for several highlights!
+
+- [broom](https://broom.tidymodels.org/news/index.html#broom-0-7-8-2021-06-24)
+- [butcher](https://butcher.tidymodels.org/news/index.html#butcher-0-1-4-2021-03-19)
+- [discrim](https://discrim.tidymodels.org/news/index.html#discrim-0-1-2-2021-05-28)
+- [hardhat](https://hardhat.tidymodels.org/news/index.html#hardhat-0-1-4-2020-07-02)
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-0-1-6-2021-05-27)
+- [probably](https://probably.tidymodels.org/news/index.html#probably-0-0-6-2020-06-05)
+- [recipes](https://recipes.tidymodels.org/news/index.html#recipes-0-1-16-2021-04-16)
+- [rsample](https://rsample.tidymodels.org/news/index.html#rsample-0-1-0-2021-05-08)
+- [stacks](https://github.com/tidymodels/stacks/blob/main/NEWS.md#v020)
+- [themis](https://themis.tidymodels.org/dev/news/index.html#themis-0-1-4-2021-06-12)
+- [tidymodels](https://tidymodels.tidymodels.org/news/index.html#tidymodels-0-1-3-2021-04-19)
+- [tidyposterior](https://github.com/tidymodels/tidyposterior/blob/master/NEWS.md#tidyposterior-010)
+- [tune](https://tune.tidymodels.org/news/index.html#tune-0-1-5-2021-04-23)
+- [workflowsets](https://workflowsets.tidymodels.org/news/index.html#workflowsets-0-0-2-2021-04-16)
+- [yardstick](https://yardstick.tidymodels.org/news/index.html#yardstick-0-0-8-2021-03-28)
```

Comment:
I don't have a hot take on this, but do you think it's worth adding the version number for each item here, since you're linking to the anchored item?

Reply:
I don't think that will add much for the reader? I think I'll pass on that.